### PR TITLE
Hello World, mkdocs / mkdocstrings

### PR DESCRIPTION
**Pull request by ssweber**
To test, you'd need to:

pip install:
mkdocs
mkdocstrings
mkdocstrings-python
mkdocs-material

then cd into pysimplesql and run:
`mkdocs serve`

I’ll start breaking things out into pages, and only include functions useful for development.

When we are ready, we can either host on GitHub pages, or read the docs.
See https://www.mkdocs.org/user-guide/deploying-your-docs/

### CODE REVIEW

**Comment by ssweber** [comment]
Let me know if you like this direction. I saw ruff was using it for their documentation.

File: logo/icon.svg
Comment (ssweber) [inline logo/icon.svg:1]: For some reason the svg I committed had a 100% white background, changed here

**Comment by ssweber** [comment]
#198

**Comment by PySimpleSQL** [comment]
Awesome thanks.  I'm done for the day today, but I will check it out first
thing tomorrow.  I'm excited to see the docs get organized

**Comment by ssweber** [comment]
Great. This commit doesn’t do much, just wanted to get a spot ready so there is a place to put more comprehensive docs. That way there won’t be so much pressure to fit it all in the Readme.

**Review by PySimpleSQL — APPROVED** [review]

**Pull request merged**